fix(tribe-commerce): Add a check for zero price to ticket validation. (#1206)
moderntribe/event-tickets: readme.txt

* Tweak - After deleting attendees you are now redirected back to the admin attendee page to clear the url of the deleting actions [122083]
* Tweak - Tribe Commerce knowledge base article link now opens up in a new window [122331]
* Fix - Filter the attendee registration display to only show tickets for the current provider. Add provider to attendee registration url [122317]

> * Fix - Do not allow $0 tickets in the block editor for Tribe Commerce - to prevent PayPal issues. [123835]

Reviewer: ```suggestion
* Fix - Prevent potential PayPal issues by not allowing $0 tickets in the block editor for Tribe Commerce [123835]
```
